interface/src/Application.cpp

```diff
@@ -5534,6 +5539,21 @@ void Application::loadSettings() {
     }
 
     getMyAvatar()->loadData();
+
+
+    auto bucketEnum = QMetaEnum::fromType<ExternalResource::Bucket>();
+    auto er = ExternalResource::getInstance();
+
+    for(int i=0;i<bucketEnum.keyCount();i++) {
+        const char *keyName = bucketEnum.key(i);
+        QString setting("ExternalResource/");
+        setting += keyName;
+        auto bucket = static_cast<ExternalResource::Bucket>(bucketEnum.keyToValue(keyName));
+        Setting::Handle<QString> url(setting, er->getBase(bucket));
+        er->setBase( bucket, url.get() );
+    }
```

Comment:
```suggestion
    for (int i = 0; i < bucketEnum.keyCount(); i++) {
        const char* keyName = bucketEnum.key(i);
        QString setting("ExternalResource/");
        setting += keyName;
        auto bucket = static_cast<ExternalResource::Bucket>(bucketEnum.keyToValue(keyName));
        Setting::Handle<QString> url(setting, er->getBase(bucket));
        er->setBase(bucket, url.get());
    }
```
And similar elsewhere.